#ICARUS GeneList
uint8 Stage
string[] GeneName
float32[] GeneValue